pkg/control/control.go

```diff
@@ -239,6 +239,8 @@ ENTRY:
 
 // RunSelfScheduled runs the controller with self scheduled
 func (c *Controller) RunSelfScheduled() {
+	c.setUpDB()
```

Comment:
This will fall into noop client, right?

Reply:
https://github.com/pingcap/tipocket/blob/51554a29ad03de1b5f7b786ff3b94ad560c7d6af/cmd/ledger/main.go#L68

because I specify a `noop` DB, c.SetUpDB() does nothing here.